pkg/runs/runsGet.go

```diff
@@ -47,7 +47,7 @@ func GetRuns(
 			if err == nil {
 
 				// Some formatters need extra fields filled-in so they can be displayed.
-				if chosenFormatter.IsNeedingDetails() {
+				if chosenFormatter.IsNeedingMethodDetails() {
 					runJson, err = GetRunDetailsFromRasSearchRuns(runJson, apiServerUrl)
```

Comment:
Why isn't the method detail available already from the original REST call ? Seems that it possibly should be returned in all cases from the query.